com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/api/impl/MtasApiServiceImpl.java

```diff
@@ -88,11 +91,14 @@ private Module getModule(DeployedMtaModule module) {
                               .moduleName(module.getModuleName())
                               .providedDendencyNames(module.getProvidedDependencyNames())
                               .uris(module.getUris())
-                              .services(module.getServices())
+                              .services(module.getResources()
+                                              .stream()
+                                              .map(s -> s.getServiceName())
```

Comment:
In the change above you use a method reference, here you don't.
Use one or the other in all places